lib/internal/modules/esm/translators.js

```diff
@@ -516,6 +519,12 @@ translators.set('wasm', async function(url, source) {
     if (impt.kind === 'global') {
       ArrayPrototypePush(wasmGlobalImports, impt);
     }
+    if (impt.module.startsWith('wasm-js:')) {
```

Comment:
Can we link to the spec where specifies throwing behavior here?

Reply:
Sure, added.